fuzzing/common.bzl

```diff
@@ -74,7 +82,7 @@ def _fuzzing_corpus_impl(ctx):
         command = "cp $@",
     )
 
-    return [DefaultInfo(files = depset([corpus_dir]))]
+    return [DefaultInfo(runfiles = ctx.runfiles(files = [corpus_dir]), files = depset([corpus_dir]))]
```

Comment:
This is getting a bit complex on one line, perhaps break it such that `runfiles = ...` and `files = ...` are on separate lines?